ckan/config/middleware/flask_app.py

```diff
@@ -246,12 +249,25 @@ def save_session(self, app: Any, session: Any, response: Any):
     _register_core_blueprints(app)
     _register_error_handler(app)
 
+    # CSRF
+    app.config['WTF_CSRF_FIELD_NAME'] = config.get(
+        'ckan.csrf_protection.field_name', "_csrf_token"
+    )
+    _csrf.init_app(app)
+
     # Set up each IBlueprint extension as a Flask Blueprint
     for plugin in PluginImplementations(IBlueprint):
         plugin_blueprints = plugin.get_blueprint()
-        if not isinstance(plugin_blueprints, list):
-            plugin_blueprints = [plugin_blueprints]
-        for blueprint in plugin_blueprints:
+        # plugin_blueprints must not send as list in csrf.exempt
+        if isinstance(plugin_blueprints, list):
+            plugin_blueprints = plugin_blueprints[0]
+        # we need to exempt CKAN extensions till they are ready
+        # to implement the csrf_token to their forms, otherwise
+        # they will get 400 Bad Request: The CSRF token is missing.
+        if not asbool(config.get("ckan.csrf_protection.enabled", False)):
+            _csrf.exempt(plugin_blueprints)
```

Comment:
I find the `ckan.csrf_protection.enabled` confusing since it is enabled by default, but this is just an exempt for extensions. I think we should make this config name more explicit to its use. 

Something like: `ckan.csrf_protection.exempt_extensions` or `ckan.csrf_protection.ignore_extensions`

Also, I would add a `log.warn()` saying that we are allowing extensions to run without CSRF protection but that It will be forced in the future.

Reply:
If fact, a good log.warn(...) can also be usefult to explain what are we explaining in the comment. (And therefore, remove it)